### DISCUSSION TITLE
The idea of adding some features

### DISCUSSION THREAD

**Discussion by N-Master789** [KadeArchive/Kade-Engine#352] [Ideas]
I have some cool ideas for the updates:
1) Mid-song difficulty changes feature
- Like your song is very long and you want to make the second half of it harder
2) Song part randomizer
- I mean like you have instrumental and voices version of starting part, that will make song always start from that part, and then other parts of song will be just randomly using parts (And also this must be connected to song chart). Also there should be settings file that will manage connection of song parts to charts and will configure other things like possible repeats of some parts and etc.

**Comment by isakube** [comment]
1. just make the second half more difficult lol
2. uhhh my brain is small and i just woke up .~.
